java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -344,8 +357,18 @@ protected final TStruct deserializeValue(TaggedDeserializationContext context) t
         return value;
     }
 
+    @Override
+    protected final TStruct deserializeValue(UntaggedDeserializationContext context) throws IOException {
+        TStruct value = this.newDefaultValue();
+        if (this.baseStructType != null) {
+            this.baseStructType.deserializeValueAsBase(context, value);
+        }
+        this.deserializeStructFields(context, value);
```

Comment:
I already made a comment that this block is identical to deserializeValueAsBase and you can just call that method here. It's different for tagged since need to call begin/end methods.